deployment/docker/README.md

```diff
@@ -36,51 +36,49 @@ VictoriaMetrics will be accessible on the following ports:
-* [alertmanager](#alertmanager) is configured to receive notifications from vmalert.
+* [alertmanager](#alertmanager) is configured to receive notifications from `vmalert`.
 
-To access `vmalert` via `vmselect`
-use link [http://localhost:8428/vmalert](http://localhost:8428/vmalert/).
+To access `vmalert` use link [http://localhost:8428/vmalert](http://localhost:8428/vmalert/).
 
 To access [vmui](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#vmui)
 use link [http://localhost:8428/vmui](http://localhost:8428/vmui).
 
 ## VictoriaMetrics cluster
 
-VictoriaMetrics cluster environemnt consists of vminsert, vmstorage and vmselect components. vmselect
-has exposed port `:8481`, vminsert has exposed port `:8480` and the rest of components are available
-only inside of environment.
+VictoriaMetrics cluster environment consists of `vminsert`, `vmstorage` and `vmselect` components.
+`vmselect` has exposed port `:8481`, `vminsert` has exposed port `:8480` and the rest of components 
+are available only inside the environment.
 
 The communication scheme between components is the following:
-* [vmagent](#vmagent) sends scraped metrics to vminsert;
-* vminsert forwards data to vmstorage;
-* vmselect is connected to vmstorage for querying data;
-* [grafana](#grafana) is configured with datasource pointing to vmselect;
-* [vmalert](#vmalert) is configured to query vmselect and send alerts state
-  and recording rules to vminsert;
-* [alertmanager](#alertmanager) is configured to receive notifications from vmalert.
+* [vmagent](#vmagent) sends scraped metrics to `vminsert`;
+* `vminsert` forwards data to `vmstorage`;
+* `vmselect` is connected to `vmstorage` for querying data;
+* [grafana](#grafana) is configured with datasource pointing to `vmselect`;
+* [vmalert](#vmalert) is configured to query `vmselect` and send alerts state
+  and recording rules to `vminsert`;
+* [alertmanager](#alertmanager) is configured to receive notifications from `vmalert`.
 
-To access `vmalert` via `vmselect` 
-use link [http://localhost:8481/select/0/prometheus/vmalert](http://localhost:8481/select/0/prometheus/vmalert/).
+To access `vmalert` use link [http://localhost:8481/select/0/prometheus/vmalert](http://localhost:8481/select/0/prometheus/vmalert/).
 
 To access [vmui](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#vmui) 
 use link [http://localhost:8481/select/0/prometheus/vmui](http://localhost:8481/select/0/prometheus/vmui).
 
 ## vmagent
 
-vmagent is used for scraping and pushing timeseries to
-VictoriaMetrics instance. It accepts Prometheus-compatible
-configuration `prometheus.yml` with listed targets for scraping.
+vmagent is used for scraping and pushing time series to VictoriaMetrics instance. 
+It accepts Prometheus-compatible configuration [prometheus.yml](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/prometheus.yml)
+with listed targets for scraping.
```

Comment:
should we mentioned that it supports only limited configuration from this file?

Reply:
This isn't the main docs for VictoriaMetrics - these are supplementary docs for docker-compose, so I think there is no sense in bloating these docs.